src/components/templates/ProductTemplate/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { ProductTemplateLoading } from './ProductTemplateLoading/ProductTemplateLoading';
+import { ProductTemplateProgress } from './ProductTemplateProgress/ProductTemplateProgress';
+import { ProductTemplateCard } from './ProductTemplateCard/ProductTemplateCard';
+import ProductTemplateComponent, { ProductTemplateProps } from './ProductTemplate/ProductTemplate';
+import React from 'react';
+
+interface ProductTemplateStatic {
+  Loading: typeof ProductTemplateLoading;
+  Progress: typeof ProductTemplateProgress;
+  Card: typeof ProductTemplateCard;
+}
+
+const ProductTemplate: ProductTemplateStatic & React.FC<ProductTemplateProps> = (props) => (
```

Comment:
I've noticed this is a trend in this repo, but there seem to be quite a few opinions against using React.FC (which is why I used the other form) - https://fettblog.eu/typescript-react-why-i-dont-use-react-fc/

Reply:
https://levelup.gitconnected.com/ultimate-react-component-patterns-with-typescript-2-8-82990c516935

Reply:
I think we should stick one way of doing things for consistency. Maybe we can create an issue in our ZSC board so someone can have a look at it and change everything.

Reply:
Fair enough - that makes complete sense